hadoop-hdds/docs/content/start/StartFromDockerHub.md

```diff
@@ -72,7 +72,7 @@ connecting to the SCM's UI at [http://localhost:9876](http://localhost:9876).
 
 The S3 gateway endpoint will be exposed at port 9878. You can use Ozone's S3
 support as if you are working against the real S3.  S3 buckets are stored under
-the `/s3v` volume, which needs to be created by an administrator first:
+the `/s3v` volume:
 
 ```
 docker-compose exec scm ozone sh volume create /s3v
```

Comment:
This code block can be removed, too.

Reply:
Thanks @bharatviswa504 for updating the patch.  I think you missed this `volume create` command.